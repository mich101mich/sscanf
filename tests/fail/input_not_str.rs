fn main() {
    sscanf::scanf!(5 + 4, "{}", usize);
}
