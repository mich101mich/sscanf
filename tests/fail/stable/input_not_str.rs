fn main() {
    sscanf::sscanf!(5usize, "{}", usize);
    sscanf::sscanf!(b"5", "{}", usize);
}
