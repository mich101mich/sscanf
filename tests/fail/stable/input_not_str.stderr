error[E0308]: mismatched types
 --> tests/fail/stable/input_not_str.rs:2:21
  |
2 |     sscanf::sscanf!(5usize, "{}", usize);
  |     ----------------^^^^^^--------------
  |     |               |
  |     |               expected `str`, found `usize`
  |     arguments to this function are incorrect
  |
  = note: expected reference `&str`
             found reference `&usize`
note: associated function defined here

error[E0308]: mismatched types
 --> tests/fail/stable/input_not_str.rs:3:21
  |
3 |     sscanf::sscanf!(b"5", "{}", usize);
  |     ----------------^^^^--------------
  |     |               |
  |     |               expected `str`, found `&[u8; 1]`
  |     arguments to this function are incorrect
  |
  = note: expected reference `&str`
             found reference `&&'static [u8; 1]`
note: associated function defined here
